src/components/grid-aware/Spacer/Spacer.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+
+import s from "./Spacer.module.css";
+
+type SpacerProps = {
+  heightDesktop: string;
+  heightMobile: string;
+  color?: string;
+};
+
+const Spacer = ({
+  heightDesktop,
+  heightMobile,
+  color = "transparent",
+}: SpacerProps) => (
+  <div
+    style={
+      {
+        "--spacer-height-desktop": heightDesktop,
+        "--spacer-height-mobile": heightMobile,
+        "--spacer-color": color,
+      } as React.CSSProperties
```

Comment:
Ah, this is interesting. I hadn't thought about how the CSS custom properties would interact with TypeScript. I think this approach is OK for CSS custom properties that are local to just that component, but if we ever find ourselves using custom properties across multiple components, then we may want to try out an approach closer to https://stackoverflow.com/a/65959390, where they defined a custom interface that extends the `CSSProperties` interface.

Reply:
Interesting, forgot about this mechanic also.